rasa/core/channels/botframework.py

```diff
@@ -244,10 +243,8 @@ async def webhook(request: Request) -> HTTPResponse:
                 else:
                     logger.info("Not received message type")
             except Exception as e:
-                logger.error(f"Exception when trying to handle message.{e}")
+                logger.error(f'Exception when trying to handle message.{e}')
                 logger.debug(e, exc_info=True)
-                pass
-
```

Comment:
Function `BotFrameworkInput.blueprint.webhook` refactored with the following changes:

- Remove redundant pass statement ([remove-redundant-pass](https://docs.sourcery.ai/refactorings/remove-redundant-pass/))